man/systemd.network.xml

```diff
@@ -4451,6 +4451,79 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[EarliestTxTimeFirst] Section Options</title>
+    <para>The [EarliestTxTimeFirst] section manages the queueing discipline (qdisc) of Earliest TxTime First
+    (etf).</para>
+
+    <variablelist class='network-directives'>
+      <xi:include href="tc.xml" xpointer="qdisc-parent" />
+      <xi:include href="tc.xml" xpointer="qdisc-handle" />
+
+      <varlistentry>
+        <term><varname>Clockid=</varname></term>
+        <listitem>
+          <para>The ETF (Earliest TxTime First) qdisc allows applications to control
+          the instant when a packet should be dequeued from the traffic control
+          layer into the netdevice.
+          Specifies the clock to be used by qdisc's internal timer for measuring
+          time and scheduling events. The qdisc expects that packets passing
+          through it to be using this same as the reference of their txtime timestamps.
+          It will drop packets coming from sockets that do not comply with that.
+          See tc-etf(8) for more details.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Delta=</varname></term>
+        <listitem>
+          <para>After enqueueing or dequeueing a packet, the ETF qdisc will schedule its
+          next wake-up time for the next txtime minus this delta value. This means
+          can be used as a fudge factor for the scheduler latency of a system.
+          This value must be specified in nanoseconds.
+          The default value is 0 nanoseconds.
+          See tc-etf(8) for more details.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Deadline=</varname></term>
+        <listitem>
+          <para> When option is set Y, the ETF disc will handle txtime with a different semantics,
```

Comment:
we allow various ways to specify booleans, but in docs we kinda want that settings are documented to accept "yes" and "no"